pymongo/collection.py

```diff
@@ -1379,6 +1379,8 @@ def create_indexes(self, indexes):
            Apply this collection's write concern automatically to this operation
            when connected to MongoDB >= 3.4.
         .. versionadded:: 3.0
+
+        .. _createIndexes: https://docs.mongodb.com/manual/indexes/
```

Comment:
According to Kay: There is no dochub link to this. Dochub links are created upon request of an engineer, so that they can put a link in their docs to a page that does not yet exist. When pages are moved in the MongoDB manual, there are redirects set up so that we avoid link rot.